realworld-OOP-lab/tests/tests.js

```diff
@@ -0,0 +1,46 @@
+import {expect} from 'chai';
+import {Iphone} from '../src/OOP-challenge.js';
+import {Samsung} from '../src/OOP-challenge.js';
+import {Microsoft} from '../src/OOP-challenge.js';
+// import Phone from '../src/OOP-challenge.js';
+
+
+
+describe("Create an Iphone class", () => {
+	it("should be an object", () => {
+		const applePhone = new Iphone();
+        expect(typeof applePhone).to.eql(typeof {});
+   
+});
```

Comment:
Be careful with Indentation, Indentation should be consistent. Using two spaces to indent is preferred

Reply:
noted.